orders/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+
+class Order(models.Model):
+    total_quantity     =   models.IntegerField(null=True)
```

Comment:
```suggestion
    total_quantity     =   models.IntegerField(default=0)
```
null True도 좋지만 어차피 숫자가 기록될 곳이라면 null 보다는 기본값 0을 주는건 어떨까요?

Reply:
수정 완료 했습니다.